cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/http/filter/flow/control/FlowControlledRequestFilter.java

```diff
@@ -38,13 +37,10 @@
 public class FlowControlledRequestFilter implements ContainerRequestFilter {
 
     private final RateLimiter restRequestRateLimiter;
-    private final CountDownLatch restAdapterStartLatch;
 
     @Inject
-    public FlowControlledRequestFilter(final RateLimiterConfiguration rateLimiterConfiguration,
-            final CountDownLatch restAdapterStartLatch) {
+    public FlowControlledRequestFilter(final RateLimiterConfiguration rateLimiterConfiguration) {
         restRequestRateLimiter = rateLimiterConfiguration.restRequestRateLimiter();
```

Comment:
This `CountDownLatch` was used to prevent processing any REST requests until a certain stage had been reached in the old blue-green deployment procedure. As this procedure is no longer used, this latch is obsolete and has been removed (configured by class `RestAdapterStartLatchConfiguration`)